### ISSUE TITLE
wine 4.8: winetricks install mdac28 stucks

### ISSUE THREAD

**Issue by d9k** [Winetricks/winetricks#1241]
```
 % lsb_release -a                     
Description:    Linux Mint 19.1 Tessa

 % echo $WINEARCH 
win32

 % winetricks --version
20180217 - sha256sum: 1b156c616174f41df79d72a90c52eb2b110c63a6e8ecb865d303a8f9f0908924

 % wine --version      
wine-4.8

 % uname -a
Linux komarov-VirtualBox 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

` % WINEDEBUG=+relay,+seh,+tid winetricks mdac28 >>/tmp/winetricks-error.log 2>&1`

In `Microsoft Data Access Components 2.8 SP 1` window I see `Installing Microsoft C/C++ Runtime libraries` process stuck and at the bottom of the log file I see the same lines repeated: 

```
0056:Call KERNEL32.CopyFileW(011830f8 L"C:\\users\\komarov\\Temp\\IXP000.TMP\\",01182880 L"C:\\Program Files\\Common Files\\Microsoft Shared\\dasetup\\msvcrt.inf",00000000) ret=7
dd4c411
0056:Ret  KERNEL32.CopyFileW() retval=00000000 ret=7dd4c411
0056:Call setupapi.SetupDefaultQueueCallbackW(01183958,0000000e,00e13da0,00e13db8) ret=7dd50af6
0056:Ret  setupapi.SetupDefaultQueueCallbackW() retval=00000001 ret=7dd50af6
0056:Call ntdll.RtlSizeHeap(00110000,00000000,011830f8) ret=7dd4b30d
0056:Ret  ntdll.RtlSizeHeap() retval=00000046 ret=7dd4b30d
0056:Call ntdll.RtlSizeHeap(00110000,00000000,01182880) ret=7dd4b333
0056:Ret  ntdll.RtlSizeHeap() retval=00000086 ret=7dd4b333
0056:Call setupapi.SetupDefaultQueueCallbackW(01183958,0000000b,00e13d90,00000000) ret=7dd50a48
0056:Ret  setupapi.SetupDefaultQueueCallbackW() retval=00000001 ret=7dd50a48
0056:Call ntdll.RtlAllocateHeap(00110000,00000000,0000006e) ret=7dd4afad
0056:Ret  ntdll.RtlAllocateHeap() retval=01183148 ret=7dd4afad
0056:Call KERNEL32.CreateDirectoryW(01183148 L"C:\\Program Files\\Common Files\\Microsoft Shared\\dasetup",00000000) ret=7dd4b00b
0056:Ret  KERNEL32.CreateDirectoryW() retval=00000000 ret=7dd4b00b
0056:Call ntdll.RtlFreeHeap(00110000,00000000,01183148) ret=7dd4b096
0056:Ret  ntdll.RtlFreeHeap() retval=00000001 ret=7dd4b096
0056:Call KERNEL32.GetFileAttributesW(01182880 L"C:\\Program Files\\Common Files\\Microsoft Shared\\dasetup\\msvcrt.inf") ret=7dd4c4b3
0056:Ret  KERNEL32.GetFileAttributesW() retval=ffffffff ret=7dd4c4b3
```

**Comment by d9k** [comment]
```
~/.wine/drive_c
 % ls -la Program\ Files/Common\ Files/Microsoft\ Shared/dasetup
total 8
drwxrwxr-x 2 komarov komarov 4096 May 24 18:39 .
drwxrwxr-x 4 komarov komarov 4096 May 24 18:39 ..
```

**Comment by austin987** [comment]
Can you try in older wine?

**Comment by d9k** [comment]
@austin987, it worked with wine 4.2 and 64-bit WINEARCH, I think.

What about wine 4.8?

**Comment by austin987** [comment]
It's almost certainly a regression in wine they should be reported upstream. If you can do a bisect of wine, that would be helpful.

**Comment by d9k** [comment]
> It's almost certainly a regression in wine they should be reported upstream. If you can do a bisect of wine, that would be helpful.

Thanks for advice, but no time for that now. I think I wouldn't use this app for mdac.

**Comment by austin987** [comment]
I can reproduce in wine-4.8, but it looks like it's already fixed in wine-4.9, so I'd recommend simply updating wine.
